Tests are broken

Seems that some files are missing for the test script to work correctly:

```shell
[william@william-pc coding-coach]$ yarn test
yarn run v1.9.4
$ node scripts/custom-config test --env=jsdom
/home/william/Dev/Self/WNemencha/coding-coach/node_modules/proxyquire/lib/proxyquire.js:161
    throw err
    ^

Error: Cannot find module '../utils/createJestConfig' from '/home/william/Dev/Self/WNemencha/coding-coach/node_modules/react-scripts/scripts'
    at Function.module.exports [as sync] (/home/william/Dev/Self/WNemencha/coding-coach/node_modules/resolve/lib/sync.js:43:15)
    at Proxyquire._resolveModule (/home/william/Dev/Self/WNemencha/coding-coach/node_modules/proxyquire/lib/proxyquire.js:137:20)
    at Proxyquire.<anonymous> (/home/william/Dev/Self/WNemencha/coding-coach/node_modules/proxyquire/lib/proxyquire.js:205:35)
    at Array.reduce (<anonymous>)
    at Proxyquire._withoutCache (/home/william/Dev/Self/WNemencha/coding-coach/node_modules/proxyquire/lib/proxyquire.js:204:6)
    at Proxyquire.load (/home/william/Dev/Self/WNemencha/coding-coach/node_modules/proxyquire/lib/proxyquire.js:129:15)
    at Object.<anonymous> (/home/william/Dev/Self/WNemencha/coding-coach/scripts/custom-config.js:39:5)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/src/config/constants.js
+++ b/src/config/constants.js
@@ -3,7 +3,7 @@ export default {
     EMAIL: process.env.REACT_APP_EMAIL || 'codingcoachio@gmail.com',
     SLACK_URL:
       process.env.REACT_APP_SLACK_URL ||
-      'https://join.slack.com/t/coding-coach/shared_invite/enQtNDMyMzUzNjAxODQyLTUwYTFkMzZmNGFhMzZjYTQwOWE2YWFjOGJhNzYyOGIxNDM5Zjc1YWQ4NjMwN2U5YzBlNjkwMTI2ZDNiOWQyMzM',
+      'https://join.slack.com/t/coding-coach/shared_invite/enQtNDY3NjI0NzYwNTEyLTA1OTNhZmI0MWNjZTdhMTdiNDc0YjZjYTg3NWU5YmU2MjVjMWUwMWQ3ZWU2Y2I5YjM1YjAyZTkzNWU3YjczOTQ',
   },
   social: {
     FB_URL: process.env.REACT_APP_FACEBOOK_URL || 'https://www.facebook.com/codingcoachio/',

